larky/src/test/resources/quick_tests/test_CTR.star

```diff
@@ -0,0 +1,33 @@
+load("@stdlib//unittest","unittest")
```

Comment:
You should absolutely port this test https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/SelfTest/Cipher/test_CTR.py to ensure 100% compatibility with pycryptodome. The quick_tests are OK to test something quickly, but for crypto items we should port the pycryptodome tests as we've done previously. See the current test suite here for ciphers: https://github.com/verygoodsecurity/starlarky/tree/master/larky/src/test/resources/vendor_tests/Crypto/Cipher or https://github.com/verygoodsecurity/starlarky/tree/master/larky/src/test/resources/vendor_tests/Crypto for the same test suites that was ported from pycryptodome.